spec/support/saml_auth_helper.rb

```diff
@@ -30,12 +30,11 @@ def saml_settings
     settings
   end
 
-  def request_authn_context
-    if IdentityConfig.store.aal_authn_context_enabled
-      Saml::Idp::Constants::AAL2_AUTHN_CONTEXT_CLASSREF
-    else
-      Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF
-    end
+  def request_authn_contexts
+    [
+      Saml::Idp::Constants::AAL2_AUTHN_CONTEXT_CLASSREF,
+      Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF,
+    ]
```

Comment:
Can you clarify: Is there expected to be a behavior change with this? If the goal is to simply remove all code paths where `aal_authn_context_enabled` could assume to ever be `false`, wouldn't this still return just the singular `Saml::Idp::Constants::AAL2_AUTHN_CONTEXT_CLASSREF`?

Reply:
There were a handful of tests that were expecting the previous default of IAL1. Without specifying IAL1 as a requested authn context, the resulting request was defaulting to IAL2, and the tests were failing because the user wasn't verified.

Reply:
Ah, I had missed that this was a spec support file.